src/Arriba/Tools/Arriba.WorkItemCrawler/ItemProviders/AzureDevOpsItemProvider.cs

```diff
@@ -127,8 +127,8 @@ private object GetFieldValue(JToken item)
 
         private async Task<ColumnDetails> GetColumnFromName(string columnName)
         {
-            var set = await this.Columns.Value;
-            return set.First(x => String.Equals(x.Name, columnName, StringComparison.OrdinalIgnoreCase));
+            var set = await Columns.Value;
```

Comment:
var set = await GetColumsAsync();

to standardize on the way to get the columns

Reply:
Agreed this should be updated, but not part of this PR. We should take another change that cleans up and refactors the crawler